Scripts/Core/EditorVR.Tools.cs

```diff
@@ -289,7 +289,8 @@ internal static bool SelectTool(Transform rayOrigin, Type toolType, bool despawn
 						
 						deviceInputModule.UpdatePlayerHandleMaps();
 						result = spawnTool;
-					}
+					} else if (hideMenu)
```

Comment:
nitpick :  the accompaning `if` for this is way above, and we generally don't cuddle elses anyways, right ?  should probably add brackets, or at least move to the next line

Reply:
Good call. I think my formatting settings might be a little wonky